app/serializers/submission-internal-review.js

```diff
@@ -0,0 +1,13 @@
+import ApplicationSerializer from './application';
+
+const FORCE_SERIALIZED = ['submissions'];
+
+export default class SubmissionInternalReviewSerializer extends ApplicationSerializer {
+  shouldSerializeHasMany(snapshot, key, relationshipType) {
+    if (FORCE_SERIALIZED.includes(key)) {
+      return true;
+    } else {
+      return super.shouldSerializeHasMany(snapshot, key, relationshipType);
+    }
+  }
+}
```

Comment:
the HasMany relation would not save without this.
I think Sergio did say something about it awhile ago.
Default one-to-many in ember does not save the many side unless forced.
Here I wanted to force it so we could only update `SubmissionInternalReview.submissions` to avoid concurrency/data loss by profiles who do not have the models in data.